pkg/ovs/ovsctl/ovsctl_others.go

```diff
@@ -47,9 +49,17 @@ func ovsVSwitchdUDS() string {
 	// are possible. Besides, this value is only used when invoking ovs-appctl (as a new
 	// process) at the moment, so the overhead of reading the PID from file should not be a
 	// concern.
-	pid, err := readOVSVSwitchdPID()
-	if err != nil {
-		klog.ErrorS(err, "Failed to read ovs-vswitchd PID")
+	var pid int
+	var readErr error
+	pollErr := wait.PollImmediate(50*time.Millisecond, 5*time.Second, func() (done bool, err error) {
+		pid, readErr = readOVSVSwitchdPID()
+		if err != nil {
+			return false, nil
```

Comment:
do you think it's worth logging something with V(3) or V(4) here?

Reply:
Then we will need to reduce the frequency of retry or use another counter for logging. Otherwise there would be 100 logs for each call in worse case. If we want to be aware of the waiting duration, how about counting the time it spent on waiting with V(0), then ideally there will be few such logs when the OVS starts or restarts?

Reply:
sounds good to me. I think we should only log it when there is an actual wait or use a higher verbosity value. I believe this function is also called pretty often by the Flow Exporter on Kind clusters.

Reply:
Done, thanks